rollbar-java/src/main/java/com/rollbar/notifier/sender/SyncSender.java

```diff
@@ -22,7 +21,7 @@
 
   public static final String DEFAULT_API_ENDPOINT = "https://api.rollbar.com/api/1/item/";
 
-  private static final String UTF_8 = StandardCharsets.UTF_8.name();
```

Comment:
The fact that the standard (correct) way to do this is `StandardCharsets.UTF_8.name()` is one the most Java things I have ever seen.